src/DETHRACE/common/opponent.c

```diff
@@ -1210,7 +1210,43 @@ void ProcessLevitate(tOpponent_spec* pOpponent_spec, tProcess_objective_command
     float terminal_time;
     float y;
     LOG_TRACE("(%p, %d)", pOpponent_spec, pCommand);
-    NOT_IMPLEMENTED();
+
+    if (pCommand == ePOC_start) {
+        dr_dprintf("%s: ProcessLevitate() - new objective started", pOpponent_spec->car_spec->driver_name);
+        pOpponent_spec->levitate_data.waiting_to_levitate = 1;
+        pOpponent_spec->car_spec->brake_force = 15.f * pOpponent_spec->car_spec->M;
+        pOpponent_spec->car_spec->acc_force = 0.f;
+        pOpponent_spec->levitate_data.time_started = gTime_stamp_for_this_munging;
+    } else if (pCommand == ePOC_run) {
+        if (pOpponent_spec->levitate_data.waiting_to_levitate) {
+            if ((BrVector3Length(&pOpponent_spec->car_spec->v) < .01f && BrVector3Length(&pOpponent_spec->car_spec->omega) < 1.f) || gTime_stamp_for_this_munging - pOpponent_spec->levitate_data.time_started > 4000) {
+                pOpponent_spec->levitate_data.waiting_to_levitate = 0;
+                pOpponent_spec->levitate_data.time_started = gTime_stamp_for_this_munging;
+                pOpponent_spec->levitate_data.initial_y = pOpponent_spec->car_spec->car_master_actor->t.t.translate.t.v[1];
+                if (pOpponent_spec->car_spec->has_been_stolen) {
+                    NewTextHeadupSlot(4, 250, 2500, -4, GetMiscString(170));
```

Comment:
It's flicking because this line says to "fliker for 2500ms with 250Hz"
Instead, it should be (*untested*):
```c
NewTextHeadupSlot(4, 4, 2500, -4, GetMiscString(170));
```

See
https://github.com/dethrace-labs/dethrace/blob/6061fd1d5f458caa5fed4545811e3a189129c2ed/src/DETHRACE/common/displays.c#L793-L797